api/mesh/v1alpha1/admin.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package kuma.mesh.v1alpha1;
+
+option go_package = "github.com/kumahq/kuma/api/mesh/v1alpha1";
+
+message Admin {
```

Comment:
nit: I think I'd name it `EnvoyAdmin`, top-level `Admin` proto is pretty "important" name